lib/Comment.php

```diff
@@ -168,7 +182,10 @@ public function children() {
 	}
 
 	/**
-	 * @param Comment $child_comment;
+	 * Add a child
+	 *
+	 * @param Comment $child_comment Comment child to be add;
```

Comment:
I’d use the fully qualified class name for the type. So instead of `Comment` we should use `Timber\Comment`:

```php
/**
 * @param Timber\Comment $child_comment Comment child to add.
 */
```